docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -1272,13 +1272,21 @@
 
                 <tr>
                     <td class="property" id="store-changelog-replication-factor">stores.<span class="store">store-name</span>.changelog.<br>replication.factor</td>
-                    <td class="default">2</td>
+                    <td class="default">stores.changelog.replication.factor.default</td>
```

Comment:
I have amended the commit to use the naming convention you prefer.  I hope that amending was the correct approach (as opposed to a second commit).  I also tried to add some documentation to indicate the the store name default was reserved for defining default store parameters.